### ISSUE TITLE
Create community redirects

### ISSUE THREAD

**Issue by avidseeker** [LemmyNet/lemmy#3048]
- [x] Did you check to see if this issue already exists?
- [x] Is this only a single feature request? Do not put multiple feature requests in one issue.
- [x] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/lemmy_support.
- [x] Is this a UI / front end issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo.

Add the ability for servers to set up redirects to other instance communities. E.g: Say lemmy.world instance doesn't want to have a lemmy_support community. It can instead create a link redirect from https://lemmy.world/c/lemmy_support to https://lemmy.ml/lemmy_support.

This can help reduce duplicated server entries IF a server instance is willing to.

**Comment by th0mcat** [comment]
This is exactly what Element does for Matrix rooms, and it would be a **huge** quality of life update for anyone searching for a specific community on Lemmy/kbin.  

To provide a real life example, I have my Matrix server at #cybersecurity:thomcat.rocks set as a local address for #cybersecurity:matrix.org.  This is listed in the room settings on #cybersecurity:matrix.org.

![image](https://github.com/LemmyNet/lemmy/assets/37344783/90132178-3bcb-4fe6-b46f-a0b83998a244)

**Comment by lionirdeadman** [comment]
Related to https://github.com/LemmyNet/lemmy/issues/3057

**Comment by dessalines** [comment]
This wouldn't be too difficult to implement, would be a good first issue.

**Comment by YoSoyRev** [comment]
Hi, I'd like to work on this issue. Is it still available? I'm planning to add a redirect_url field to the community table and handle the redirect at the API level ✌️

**Comment by Nutomic** [comment]
@YoSoyRev Yes you're welcome to work on this, and your proposed approach sounds reasonable. If you need any help feel free to ask in https://matrix.to/#/#lemmydev:matrix.org
